api/firmware/device.go

```diff
@@ -261,7 +293,7 @@ func (device *Device) Init() error {
 
 	// Before 2.0.0, unlock was invoked automatically by the device before USB communication
 	// started.
-	if device.version.AtLeast(semver.NewSemVer(2, 0, 0)) {
+	if device.version.AtLeast(semver.NewSemVer(2, 0, 0)) && (device.status != StatusUnlocked) {
```

Comment:
again, unrelated change and not needed. why did you add it?

Reply:
removed it now, mb I think I just thought since we can check for that now it makes sense to do so.